Model/ModelManager.php

```diff
@@ -376,7 +386,7 @@ public function getNormalizedIdentifier($entity)
         }
 
         // the entities is not managed
-        if (!$entity /*|| !$this->getEntityManager($entity)->getUnitOfWork()->isInIdentityMap($entity) // commented for perfomance concern */) {
+        if (!$entity || !$this->getEntityManager($entity)->getUnitOfWork()->isInIdentityMap($entity)) {
```

Comment:
@Soullivaneuh I've created a blackfire benchmark but I can't a real change in perfomance so I don't know why it was commented (by a bug report or ..). 

https://blackfire.io/profiles/compare/ceb1e796-d4bf-4c78-a9ab-0208df1f5fd8/graph